src/d3a/models/strategy/__init__.py

```diff
@@ -646,14 +647,16 @@ def update_offer_rates(self, market: "OneSidedMarket", updated_rate: float) -> N
             return
 
         for offer, iterated_market_id in self.offers.open.items():
+            updated_price = round(offer.energy * updated_rate, 10)
```

Comment:
This would change the behaviour and would make the updated price differences more granular as the precision in `limit_float_precision` is 8. I vote for keeping it like this.

Reply:
I would prefer to keep the precision level to global, unless in this specific case where changing precision level from 10 to the global one (8) revealed any adverse affect

Reply:
I ran all tests locally and they are passing even with precision level 8.
I will change it now but fear some tickets from researchers.